modules/edu.gemini.seqexec.web/edu.gemini.seqexec.web.client/src/main/scala/edu/gemini/seqexec/web/client/components/QueueArea.scala

```diff
@@ -51,7 +50,7 @@ object QueueTableBody {
 
   def showSequence(p: Props, s: SequenceInQueue): Callback =
     // Request to display the selected sequence
-    p.sequences.dispatchCB(NavigateTo(InstrumentPage(s.instrument, s.id.some))) >> p.sequences.dispatchCB(SelectIdToDisplay(s.id))
+    p.sequences.dispatchCB(NavigateTo(SequencePage(s.instrument, s.id, 0))) >> p.sequences.dispatchCB(SelectIdToDisplay(s.id))
```

Comment:
I am guessing the `0` here means the first step, which works only because we are using the step position as its unique id. But at some point we should change it to use the `Step` id type used by the ODB, and also use the `Location` concept (introduced in gem ?). Would this be a good time ?

Reply:
SEQNG-413 requires including the step in the url, that's what this comes from. At the moment this is unused so I leave it at 0.

I'm not aware of `Location` in gem

Reply:
That's something @swalker2m mentioned.